learn/protocol/validators/skip-blocks.md

```diff
@@ -1,43 +1,43 @@
 # Skip blocks
 
-Our consensus protocol adapts to the presence of validators who intend to [act maliciously](/learn/protocol/punishments.md) and disrupt the standard behavior of the blockchain. A single validator is enough to delay block production. In Nimiq consensus algorithm, when a validator doesn't produce a micro block in the expected time, any validator can add a skip block to the chain.
+Skip blocks are a special type of [micro block](/learn/protocol/block-format.md#micro-blocks) in the consensus protocol. When a validator fails to produce a block within the expected timeframe, the remaining validators can collectively agree to add a skip block instead of the missing micro block. This prevents delays and ensures the blockchain progresses smoothly. Once a supermajority of validators agrees to add the skip block, the chain resumes its regular progression.
```

Comment:
```suggestion
Skip blocks are a special type of [micro block](/learn/protocol/block-format.md#micro-blocks) in the consensus protocol. When a validator fails to produce a block within the expected timeframe, the remaining elected validators can collectively agree to add a skip block instead of the missing micro block. This prevents delays and ensures the blockchain progresses smoothly. Once a supermajority of elected validators agrees to add the skip block, the chain resumes its regular progression.
```